.github/workflow/release.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: Release
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
can we take this opportunity to rename `master` to `main` ?

Reply:
Yep, but first we need change repo main branch from `master` to `main`

Reply:
Right. I don't have permissions for that either